src/tracing/builder/parity.rs

```diff
@@ -122,7 +122,16 @@ impl ParityTraceBuilder {
         self,
         info: TransactionInfo,
     ) -> impl Iterator<Item = LocalizedTransactionTrace> {
-        self.into_transaction_traces_iter().map(move |trace| {
+        self.into_localized_transaction_traces_iter_with_masking(info, true)
+    }
+
+    /// Returns an iterator over all recorded traces  for `trace_transaction` with masking
+    pub fn into_localized_transaction_traces_iter_with_masking(
```

Comment:
to be consistent with the rest of the codebase, let's use "shielding"/"shield" instead of masking/mask